client/src/components/PopoverField/PopoverField.scss

```diff
@@ -1,24 +1,4 @@
 // Popover: Custom styles built on top of the Bootstrap popover component
-.popover-container {
-  position: relative;
-
-  button.btn-secondary:not(:disabled) {
-    position: absolute;
-    top: 0;
-    left: 0;
-    height: 100%;
-    width: 100%;
-    background: transparent;
-
-    &:hover,
-    &:focus,
-    &:active {
-      background-color: transparent;
-      border: 0;
-    }
-  }
-}
-
 .popover {
```

Comment:
This was added based on @flamerohr 's feedback:

> you're going to break styling things like button groups with that container logic... Another option is to move the styling to the container div and make that div look like a button (with the generic btn class from bootstrap)
Then have a transparent button cover the entire container div.

What has changed that we now don't have to do that?

Reply:
For now it looks and works appropriately; It's admittedly a less than perfect solution, until `<React.Fragment>` is available in framework. Once it is, we should set the container to the button group holding the actions, and we can stop using the wrapping div.

Reply:
Ok, so we're on react 15.3.1, and we need at least react 16.2 to use fragments. I guess we wait until then to implement it?

Reply:
Also I had to remove the extra styling because the button appeared twice.

Reply:
transparent overlay is what I described for the "real" button should be styled as :) a simple `visibility: hidden`